frontend/src/loadPostHogJS.tsx

```diff
@@ -66,6 +66,7 @@ export function loadPostHogJS(): void {
                 autocapture: {
                     capture_copied_text: true,
                 },
+                capture_performance: { web_vitals: true },
```

Comment:
IIRC these are additive so it's safe to add only web_vitals here but would be good to double check we're not turning off network capture by doing this 🙈

Reply:
This is the code

![image](https://github.com/user-attachments/assets/1b40c855-c794-4a77-adb1-a4471053d585)

So, my understanding is that this will eval to

```typescript
const clientConfigForPerformanceCapture = undefined
const networkTimingEnabled = !!networkPayloadCapture_server_side?.capturePerformance
```

So, it will fallback to whatever we've set in our admin app, aka, current behaviour. Would you agree?

Reply:
Yes perfect, thank you ! I was too lazy to check 🤣